tests/unit/challenges/test_aws_utils.py

```diff
@@ -0,0 +1,214 @@
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge,
+            self.client_token)
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge2,
+            self.client_token)
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge3,
+            self.client_token)
+
+        pk_list = [self.challenge.pk, self.challenge2.pk, self.challenge3.pk]
+        queryset = super(TestDeleteWorkers, self).queryset(pk_list)
+
+        count = 3
+        expected_workers = [None, None, None]
+        failures = []
+        expected_response = {"count": count, "failures": failures}
+        response = aws_utils.delete_workers(queryset)
+        assert response == expected_response
+        assert list(c.workers for c in queryset) == expected_workers
+
+    def test_delete_workers_when_challenge1_has_no_workers(self):
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge2,
+            self.client_token)
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge3,
+            self.client_token)
+
+        pk_list = [self.challenge.pk, self.challenge2.pk, self.challenge3.pk]
+        queryset = super(TestDeleteWorkers, self).queryset(pk_list)
+
+        message = "Please select challenges with active workers only."
+
+        count = 2
+        expected_workers = [None, None, None]
+        failures = [{"message": message, "challenge_pk": self.challenge.pk}]
+        expected_response = {"count": count, "failures": failures}
+        response = aws_utils.delete_workers(queryset)
+        assert response == expected_response
+        assert list(c.workers for c in queryset) == expected_workers
+
+    @mock.patch("challenges.aws_utils.delete_service_by_challenge_pk")
+    def test_delete_workers_exception(self, mock_delete_service_by_pk):
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge,
+            self.client_token)
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge2,
+            self.client_token)
+        aws_utils.create_service_by_challenge_pk(
+            self.ecs_client,
+            self.challenge3,
+            self.client_token)
+
+        pk_list = [self.challenge.pk, self.challenge2.pk, self.challenge3.pk]
+        queryset = super(TestDeleteWorkers, self).queryset(pk_list)
+
+        message = "errMessage"
+        exception_message = {"Error": message, "ResponseMetadata": {"HTTPStatusCode": HTTPStatus.NOT_FOUND}}
+        mock_delete_service_by_pk.return_value = exception_message
+
+        count = 0
+        failures = [
+            {"message": message, "challenge_pk": self.challenge.pk},
```

Comment:
Create a new variable called 'expected_message` with the same value as `message` and use that here.  For the sake of clarity. Do the same in other places.